src/encoded/static/components/regulome_search.js

```diff
@@ -694,10 +1063,12 @@ class RegulomeSearch extends React.Component {
             const biosampleMap = {};
             const assayMap = {};
             const targetMap = {};
+            const organMap = {};
             experimentDatasets.forEach((dataset) => {
                 biosampleMap[dataset.dataset] = dataset.biosample_ontology.term_name || '';
                 assayMap[dataset.dataset] = dataset.method || '';
                 targetMap[dataset.dataset] = dataset.targets ? dataset.targets.join(', ') : '';
+                organMap[dataset.dataset] = (dataset.biosample_ontology.classification === 'tissue') ? dataset.biosample_ontology.organ_slims.join(', ') : dataset.biosample_ontology.cell_slims.join(', ');
```

Comment:
This is the line where we sometimes add `organ_slims` and sometimes add `cell_slims`.